packages/pdfrx/lib/src/widgets/interactive_viewer.dart

```diff
@@ -398,6 +447,13 @@ class InteractiveViewer extends StatefulWidget {
   // This value was eyeballed to give a feel similar to Google Photos.
   static const double _kDrag = 0.0000135;
 
+  /// GlobalKey to access the InteractiveViewer state for animation control
+  static final GlobalKey<_InteractiveViewerState> _globalKey = GlobalKey<_InteractiveViewerState>();
```

Comment:
It's too bad to use static GlobalKey on InteractiveViewer. It makes InteractiveViewer almost singleton.